### DISCUSSION TITLE
Generator loss don't decrease in initialization training phase

### DISCUSSION THREAD

**Discussion by XuHangkun** [TobiasSunderdiek/cartoon-gan#4] [Q&A]
In initialization, I only use the content loss, but the generator loss still doesn't decrease. After the initialization training phase, the generator creates picture like this, does anyone meet this problem?
![image](https://user-images.githubusercontent.com/53259333/105031361-3079a100-5a90-11eb-8964-5d56fe0931a9.png)

**Comment by TobiasSunderdiek** [comment]
Hi,
first of all, thank you for starting this discussion and sorry for my late reply!
I did a re-run of the notebook and maybe the following observations may help you:

Epoch 05/210: Generator loss of 338023.4688
Epoch 10/210: Generator loss of  210837.703125
Epoch 11/210: Generator loss of 5.4283552169799805

The images look like the original photo images after the initalization and start to get cartoonized over the 210 epochs in my observation.

I know, this is not and answer to your question but hopefully helps in some way.

Best regards!

**Comment by keerthilogesh** [comment]
> In initialization, I only use the content loss, but the generator loss still doesn't decrease. After the initialization training phase, the generator creates picture like this, does anyone meet this problem? ![image](https://user-images.githubusercontent.com/53259333/105031361-3079a100-5a90-11eb-8964-5d56fe0931a9.png)

Am facing the same issue. Did u find the solution ? can you please explain?
